experimental/FTheoryTools/src/FamilyOfG4Fluxes/properties.jl

```diff
@@ -103,12 +215,59 @@ A family of G4 fluxes:
   - Elementary quantization checks: satisfied
   - Verticality checks: satisfied
   - Non-abelian gauge group: not broken
+  - Tadpole constraint: not analyzed
 
 julia> breaks_non_abelian_gauge_group(gf3)
 false
+
+julia> m1 = matrix_integral(gf3);
+
+julia> m2 = matrix_rational(gf3);
+
+julia> gf4 = family_of_g4_fluxes(qsm_model, m1, m2, check = false)
+A family of G4 fluxes:
+  - Elementary quantization checks: not executed
+  - Verticality checks: not executed
+  - Non-abelian gauge group: breaking pattern not analyzed
+  - Tadpole constraint: not analyzed
+
+julia> breaks_non_abelian_gauge_group(gf4, check = false)
+false
 ```
 """
-function breaks_non_abelian_gauge_group(fgs::FamilyOfG4Fluxes)
-  @req has_attribute(fgs, :breaks_non_abelian_gauge_group) "Cannot (yet) tell if this family of G4-fluxes breaks the non-abelian gauge group"
-  return get_attribute(fgs, :breaks_non_abelian_gauge_group)
+function breaks_non_abelian_gauge_group(fgs::FamilyOfG4Fluxes; check::Bool = true)
```

Comment:
```suggestion
@attr Bool function breaks_non_abelian_gauge_group(fgs::FamilyOfG4Fluxes; check::Bool = true)
```
analogous changes needed in the body